### DISCUSSION TITLE
Running into issue with Optical Flow Computations

### DISCUSSION THREAD

**Discussion by wordbrew** [volotat/SD-CN-Animation#24] [Q&A] [answered]
I am successfully cloning this repo and RAFT, installing required dependencies, and then downloading the required models, but when running the script to execute Optical Flow Computations through compute_flow.py, I keep throwing this error:

```
"0% 1/1196 [00:00<02:15,  8.81it/s]/usr/local/lib/python3.9/dist-packages/torch/functional.py:504: UserWarning: torch.meshgrid: in an upcoming release, it will be required to pass the indexing argument. (Triggered internally at ../aten/src/ATen/native/TensorShape.cpp:3483.)
  return _VF.meshgrid(tensors, **kwargs)  # type: ignore[attr-defined]
qt.qpa.xcb: could not connect to display 
qt.qpa.plugin: Could not load the Qt platform plugin "xcb" in "/usr/local/lib/python3.9/dist-packages/cv2/qt/plugins" even though it was found.
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: xcb."
```

I have tried reinstalling the application, but it does not fix the issue. Any tips (beyond recommending I wait for the extension, I'm a bit far into this and want to see success here)?

Thanks!

**Comment by volotat** [comment]
Based on the error message I'm guessing that there is a problem with Qt library that opencv relies on. You could try reinstalling the Qt library and its dependencies, or delete everything related to opencv and reinstalling it instead. Btw, do you have any graphical interface on the system you are trying to run the script?

**Reply by wordbrew** [reply]
Hi. I have tried uninstalling QT library and reinstalling, but haven't tried that with opencv. Will attempt. I don't have a graphical interface set up, beyond a form for inputs. Wasn't sure if more was needed since this was pre-web-ui stage.

**Reply by volotat** [reply, marked as answer]
There are some cv window showing current progress that might work incorrectly without a graphical interface. There are VISUALIZE parameter in the scripts, try to set it to False and run the script again. This might also negotiate the issue.
